src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -21,8 +23,11 @@ public AuthenticationController(AuthenticationService authService,
     }
 
     @PostMapping("/register")
-    public UserResponseDto register(@RequestBody UserRequestDto requestDto) {
-        User user = authService.register(requestDto.getEmail(), requestDto.getPassword());
-        return userDtoResponseMapper.mapToDto(user);
+    public UserResponseDto register(@RequestBody @Valid UserRequestDto requestDto) {
+        if (requestDto.getPassword().equals(requestDto.getRepeatPassword())) {
```

Comment:
you should do this using custom annotation, that you should use in UserRequestDto
<img width="644" alt="Screenshot 2022-10-10 at 08 39 21" src="https://user-images.githubusercontent.com/60258373/194804329-1cd6e78c-30f9-41b3-95d9-275aa39a796e.png">